src/coreservices.h

```diff
@@ -22,6 +22,8 @@ class VinylControlManager;
 class TrackCollectionManager;
 class Library;
 class LV2Backend;
+class MixxxMainWindow;
```

Comment:
revert

Reply:
It is used for the window control in mpris.